app/Http/Controllers/ParentController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\ParentModel;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class ParentController extends Controller
+{
+    public function index(Request $request)
```

Comment:
is dit een index functie of zou Search een meer toepasselijke naam zijn?
index is vaak de view van de in dit geval parentpage laten zien, maar hier ben je eigelijk al aan het reageren op user input. echter kan ik iets over het hoofd zien natuurlijk